doc/src/bdist_msi.rst

```diff
@@ -62,6 +62,8 @@ command:
        and an executable. Additional allowed keys are `argument` to specify
        the invocation of the executable, `mime` for the extension’s mime type,
        and `context` for the context menu text.
+   * - .. option:: license-file
```

Comment:
license_file

Reply:
done in [`9c3d270` (#2472)](https://github.com/marcelotduarte/cx_Freeze/pull/2472/commits/9c3d2700529ac1df700b9ea3da782812a625e811)